example-overrides/local-overrides-centos8-tripleo-train.yaml

```diff
@@ -0,0 +1,10 @@
+standalone_host: <standalone FQDN>
+# Both public_api and control_plane_ip have to be the same in OSP16 for Pacemaker
+public_api: <IP address used to reach the node>
+control_plane_ip: <Same IP address used to reach the node>
```

Comment:
I don't think this is correct. I'm not entirely surely sure how control_plane_ip is used by OOO, tbh. The big one afaik is:

```
sudo openstack tripleo deploy \
    --templates \
    --local-ip={{ control_plane_ip }} \
    --control-virtual-ip {{ public_api }} \
```

I have never looked under the covers at what --local-ip actually sets, but I do know that we add it to br-ctlplane. I don't think we want the same IP added to both br-ctlplane and br-external. The standalone example used 192.168.24.1. Do you have a specific reason for wanting to change this?

Reply:
Yeah we discussed about it (btw I copy pasted from the OSP16 local overrides example), and this is something we need to investigate further.
This has to do with Pacemaker deploying in Train, I don't remember all the details yet.

Reply:
As I say, I didn't really understand this stuff when I added it.

It sounds like we should remove control_plane_ip, which suggests we can also remove br-ctlplane. We'd then define deploy with local-ip and control-virtual-ip both using `public_api`?

Reply:
I think this isn't related to this PR, I just provided an example of an override based on OSP16 but for upstream deployment.
Yes we should fix that problem, but not in this PR imo.
Note that we don't need this hack on master.